PocketKit/Sources/PocketKit/Refresh/UserRefreshCoordinator.swift

```diff
@@ -34,6 +34,7 @@ class UserRefreshCoordinator: RefreshCoordinator {
     func refresh(isForced: Bool = false, _ completion: @escaping () -> Void) {
         Task {
```

Comment:
capturing self explicitly might not be strictly required if we are sure that the task completes (although a weak capture is a failsafe that always ensures that self is not retained more than it should), but curious if there's a specific reason why it's not captured here as opposed to other places where it's captured?

Reply:
i dumb.